docs/nimble-spright-requirements-adr.md

```diff
@@ -0,0 +1,95 @@
+
+While Nimble contributors may opt to create their component as ["incubating"](https://github.com/ni/nimble/blob/a9ba0b6027479fe1cc2267f11957caa329910dfc/packages/nimble-components/CONTRIBUTING.md#marking-a-component-as-incubating), it is expected that an incubating component is on the path to becoming a first-class Nimble component and will eventually meet all the necessary requirements. Though an incubating component can have a lower initial development cost by not having a feature complete implementation, the spec and implementation reviews may still be take considerable time / effort for the developer and the Nimble code owners.
+
+## Decision
+
+- The Nimble repo will begin hosting "Spright" packages that are not part of the Nimble Design System.
+    - These packages will parallel their Nimble counterparts: `@ni/spright-components`, `@ni/spright-angular`, and `SprightBlazor`.
+    - `spright-components` are characterized as components that are: "[molecules](https://atomicdesign.bradfrost.com/chapter-2/)", product-specific, data-connected, or not "general utility". The primary component implementation should be built on Nimble technologies (FAST, native web components, etc).
+    - The framework specific libraries, i.e. `@ni/spright-angular` and `SprightBlazor`, should contain appropriate wrappers / patterns of exposing the `spright-component` implementation in addition to framework-specific concepts used in conjunction with the components, for example Angular Pipes or  Blazor Tag Helpers.
+      - Notably, Spright is intended for components leveraging the Nimble technology stack. Spright is *NOT* intended as a place to host arbitrary Angular or Blazor components that are not implemented as web components via FAST.
+- The Nimble and Spright packages will be part of the same build, use the same infrastructure, and share the same dependency version management. Spright packages will depend on the Nimble packages.
+- Code in Spright packages will be co-owned and reviewed by the Design System team and the contributing team.
+  - The goal of spright contributions is to be technically rigorous. To follow modern HTML, CSS, and TypeScript coding styles and follow Nimble's Contributing guidelines, CSS Best Practices, naming conventions, token usage, etc.
+  - Nimble developers remain as co-owners to enforce technical quality of implementation. Long-term components live in the Nimble monorepo and have to be patched to follow common patterns, updated with dependencies, supported on test infrastructure, and continually released. This is only mantainable when all code meets the same technical implementation bar.
```

Comment:
@m-akinc @jattasNI I pushed some changes to be crisper on what I understand as our alignment on spright ownership. Made changes to context and decision  but the consequences, risks, and conformance looked spot on. If we have concerns / misalignment on the updates lets chat Friday morning quickly to resolve.

Reply:
LGTM